scraper.rb

```diff
@@ -6,11 +6,17 @@
 
 
 column_header = 'النائب'
-ar_names = EveryPolitician::Wikidata.wikipedia_xpath( 
+ar_names = EveryPolitician::Wikidata.wikipedia_xpath(
   url: 'https://ar.wikipedia.org/wiki/مجلس_النواب_اللبناني',
   after: '//h2[contains(.,"2009")]',
   xpath: '//table[.//td[contains(.,"%s")]]//td[2]//a[not(@class="new")]/@title' % column_header,
 )
-en_names = EveryPolitician::Wikidata.morph_wikinames(source: 'tmtmtmtm/lebanese-parliament-2009', column: 'wikiname')
 
-EveryPolitician::Wikidata.scrape_wikidata(names: { en: en_names, ar: ar_names }, output: false)
+en_names  = EveryPolitician::Wikidata.morph_wikinames(source: 'tmtmtmtm/lebanese-parliament-2009', column: 'wikiname')
+
+en_names2 = EveryPolitician::Wikidata.wikipedia_xpath(
+  url: "https://en.wikipedia.org/wiki/Category:Members_of_the_Parliament_of_Lebanon",
+  xpath: '//div[contains(@class, "mw-category-group")]//li/a[not(@class="new")]/@title'
+)
```

Comment:
Ah, you don't need to scrape the page directly like this. You just need:

```
en_names2 = WikiData::Category.new( 'Category:Members of the Parliament of Lebanon', 'en').member_titles
```

See docs at https://github.com/everypolitician/wikidata-fetcher